backend/src/contaxy/operations/auth.py

```diff
@@ -279,7 +279,7 @@ def create_user(
         pass
 
     @abstractmethod
-    def get_user(self, user_id: str) -> User:
+    def get_user(self, user_id: str) -> UserPermission:
```

Comment:
This change would also needs to be reflected in the client class, which is also implementing this interface